### ISSUE TITLE
Launch from CLI

### ISSUE THREAD

**Issue by klpanagi** [maciek-slon/rapp_learn_object#1]
Hi @maciek-slon ,
is it possible to launch to application without interraction with the store (?), as we are investigating MAJOR issues from the rapp-store side. Did you manage to build this RApp from the rapp-store?

We initialize the new NAO-Core-Agent, which waits for keywords and then from another terminal we execute the `LearnObject` executable but NAO does not interract.

**Comment by maciek-slon** [comment]
Yup, give me a second to prepare instructions. All scripts are already in the repository, it is just a matter of launching different set of nodes :)

**Comment by maciek-slon** [comment]
There are two options possible. 1) Run only NAOqi wrappings on the robot and then run apps by hand from the terminal, 2) Run whole Core Agent but with FakeStore, which serves RApps that are already on the robot. First one is faster, so (you will need quite a few terminals):

```bash
source /home/nao/ws_rapp_nao/install_isolated/setup.bash
roslaunch rapp_ros_naoqi_wrappings full_navigation_agent.launch
```

```bash
source /home/nao/ws_rapp_nao/install_isolated/setup.bash
rosrun rapp_ros_naoqi_wrappings acore_communication_server.py
```

At this state robot is ready to accept calls from `robots-api`, so, in yet another terminal, type:

```bash
source /home/nao/ws_rapp_nao/install_isolated/setup.bash
cd <directory with RApp executable>
./LearnObject
```

As far as I remember this should work. I will try to update files to make second option also possible (there is a FakeStore already, but it needs to have also a fake "update" method added).
